src/main/kotlin/dev/arbjerg/ukulele/jda/CommandContext.kt

```diff
@@ -42,14 +42,17 @@ class CommandContext(
 
     fun reply(msg: String) {
         channel.sendMessage(msg).queue()
+        player.lastChannel = channel
     }
 
     fun replyMsg(msg: Message) {
         channel.sendMessage(msg).queue()
+        player.lastChannel = channel
     }
 
     fun replyEmbed(embed: MessageEmbed) {
         channel.sendMessage(embed).queue()
+        player.lastChannel = channel
```

Comment:
I don't think we should set this property from this class because:

1. Non-music commands will move the channel and
2. The assignment in the play command makes this redudant